pinc/NonactivatedUser.inc

```diff
@@ -80,8 +80,7 @@ class NonactivatedUser
             FROM non_activated_users
             WHERE $field=$escaped_value
         ";
-
-        $result = mysqli_query(DPDatabase::get_connection(), $sql);
+        $result = DPDatabase::query($sql, FALSE);
```

Comment:
We should probably get rid of the error handling here and let the ::query function take care of it.

Reply:
Done. Nothing was relying on the old exception type anyway.